loadbalancer/fadein_test.go

```diff
@@ -190,6 +196,95 @@ func TestFadeIn(t *testing.T) {
 	testFadeIn(t, "consistent-hash, 9", newConsistentHash, fadeInDuration/2, fadeInDuration/3, fadeInDuration/4)
 }
 
+func closeToZero(arg int, max int) bool {
+	return arg < max/100
+}
+
+func closeToMax(arg int, max int) bool {
+	return arg > 90*max/100 && arg <= max
+}
+
+func testFadeInLoadBetweenOldEps(
+	t *testing.T,
+	name string,
+	algorithm func([]string) routing.LBAlgorithm,
+	endpointAges ...time.Duration,
+) {
+	t.Run(name, func(t *testing.T) {
+		old := 2 * fadeInDurationHuge
+		new := time.Duration(0)
+		ctx, eps := initializeEndpoints(endpointAges, fadeInDurationHuge)
+
+		a := algorithm(eps)
+		rnd := rand.New(rand.NewSource(time.Now().UnixNano()))
+		nReqs := map[string]int{}
+		stop := time.After(3 * time.Second)
```

Comment:
You had an idea than "3 * time.Second" is less that "24 * time.Hour".
So, it makes sense to move both constants to the beginning of this file and explicitly tell that
one constant << another constant there.